### PR TITLE
Add support set selection_method for group modification

### PR DESCRIPTION
**Pull request by wenyingd**

### CODE REVIEW

File: ofctrl/fgraphGroup.go [hunk 1]
```diff
@@ -109,6 +118,16 @@ func (self *Group) getGroupModMessage(command int) *openflow15.GroupMod {
 		groupMod.Type = openflow15.GT_FF
 	}
 
+	if command == openflow15.OFPGC_DELETE {
+		return groupMod
+	}
+
+	if command == openflow15.OFPGC_ADD || command == openflow15.OFPGC_MODIFY {
```
Comment (hongliangl) [inline ofctrl/fgraphGroup.go:125]: Is there something wrong to set groupMod.Properties when command is openflow15.OFPGC_DELETE?

File: ofctrl/ofctrl_test.go [hunk 2]
```diff
@@ -950,13 +950,82 @@ func TestBundle2(t *testing.T) {
...
+		param          uint64
+		fields         []openflow15.MatchField
+		propertyString string
+	}{
+		{
+			name:           "dp_hash",
+			groupId:        30,
+			mt:             openflow15.NTR_DP_HASH,
+			param:          0x1ffffffff,
+			propertyString: "selection_method=dp_hash,selection_method_param=8589934591",
```
Comment (hongliangl) [inline ofctrl/ofctrl_test.go:978]: What does the value `0x1ffffffff` represent?

File: ofctrl/fgraphGroup.go (hunk 1 above)
Comment (wenyingd) [reply ofctrl/fgraphGroup.go:125]: Yes, OVS will return error message if groupMod.Properties is not nil in deletion command

File: ofctrl/ofctrl_test.go (hunk 2 above)
Comment (wenyingd) [reply ofctrl/ofctrl_test.go:978]: No special meaning, just use it to test if "method_param" can be set as expected or not.

**Comment by wenyingd** [comment]
Close it as the PR is not updated when I push changes to the branch in my personal repo

**Comment by wenyingd** [comment]
I create a new one https://github.com/antrea-io/ofnet/pull/66 to update the latest code.

**Pull request closed without merging**